src/assets/stylesheets/ui-root.scss

```diff
@@ -223,6 +223,253 @@ body.vr-mode {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/eww-off.png);
+  }
+}
+
+:local(.iconEmoji.eww.active) {
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/eww-on.png);
+}
+:local(.iconEmoji.eww.active:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/eww-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/eww-on.png);
+  }
+}
+
+:local(.iconEmoji.disgust) {
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/disgust-off.png);
+}
+:local(.iconEmoji.disgust:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/disgust-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/disgust-off.png);
+  }
+}
+
+:local(.iconEmoji.disgust.active) {
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/disgust-on.png);
+}
+:local(.iconEmoji.disgust.active:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/disgust-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/disgust-on.png);
+  }
+}
+
+:local(.iconEmoji.hearts) {
+
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/hearts-off.png);
+}
+:local(.iconEmoji.hearts:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/hearts-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/hearts-off.png);
+  }
+}
+
+:local(.iconEmoji.hearts.active) {
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/hearts-on.png);
+}
+:local(.iconEmoji.hearts.active:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/hearts-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/hearts-on.png);
+  }
+}
+
+:local(.iconEmoji.surprise) {
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/surprise-off.png);
+}
+:local(.iconEmoji.surprise:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/surprise-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/surprise-off.png);
+  }
+}
+
+:local(.iconEmoji.surprise.active) {
+  background-image: url(../images/sprites/chest-emojis/emoji-icons/surprise-on.png);
+}
+:local(.iconEmoji.surprise.active:hover) {
+  @media (pointer: fine) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/surprise-on-hover.png);
+  }
+
+  @media (pointer: coarse) {
+    background-image: url(../images/sprites/chest-emojis/emoji-icons/happy-on.png);
+  }
+}
```

Comment:
Would be nice if all the iconEmoji CSS was moved to its own file, so it doesn't clutter ui-root.scss